server/services/status-history-logger.js

```diff
@@ -41,6 +42,23 @@ var storeLog = function (storage, logPath/*, options */) {
 	};
 };
 
+var saveRejectionReason = function (event) {
+	var status;
+	if (event.type !== 'direct') return;
+	if (!event.path.startsWith('processingSteps')) return;
+	status = unserializeValue(event.data.value);
+	if (status !== 'rejected' && status !== 'sentBack') return;
+
+	return queryMemoryDb([event.ownerId], 'businessProcessRejectionReasons', {
+		businessProcessId: event.ownerId
+	}).done(function (reasonObject) {
+		//TODO here should be added logic for saving the reasonObject
+		console.log(JSON.stringify(reasonObject));
```

Comment:
OK, mongo is already there so let's use it! https://github.com/egovernment/eregistrations/pull/1833